app/assets/javascripts/discourse/initializers/svg-sprite-fontawesome.js.es6

```diff
@@ -0,0 +1,11 @@
+import svgSpriteLoader from "discourse/lib/svg-sprite-loader";
+
+export default {
+  name: "svg-sprite-fontawesome",
+
+  initialize() {
+    if (Discourse && Discourse.SvgSpritePath) {
```

Comment:
This line means the SVG sprite isn't loaded when testing (without it, the tests would be inundated with `Unhandled request in test environment` errors). Should it be added as a fixture? Should it check for `Ember.testing` instead?